### ISSUE TITLE
Buffalo Command Update [Docs]

### ISSUE THREAD

**Issue by NlaakStudios** [gobuffalo/buffalo#1173]
## Steps to Reproduce the Problem

  Please describe in painful detail what you did (so others can play along with you) to get to this point. This includes things like the exact command(s) you used, or the `curl` command you used, that sort of thing.

Run buffalo command [cmd] --help

## Expected Behavior
Detailed documentation with no grammar or spelling errors

What did you what to happen? Tell us a story. We love to read.
Grammer, Spelling errors, as well as some long more detailed help, is missing on some commands

## Actual Behavior

Proper Grammar, Spelling and more Detailed explanations.

## Info

Please run `buffalo info` and paste the information below where it says "PASTE_HERE".

<details>
### Buffalo Version
v0.12.3

### App Information
Pwd=D:\GO_Projects\src\github.com\buffalo
Root=D:\GO_Projects\src\github.com\buffalo
GoPath=D:\GO_Projects
Name=buffalo
Bin=bin\buffalo.exe
PackagePkg=github.com/buffalo
ActionsPkg=github.com/buffalo/actions
ModelsPkg=github.com/buffalo/models
GriftsPkg=github.com/buffalo/grifts
VCS=git
WithPop=false
WithSQLite=false
WithDep=true
WithWebpack=false
WithYarn=false
WithDocker=true
WithGrifts=true

### Go Version
go version go1.10.3 windows/amd64

### Go Env
set GOARCH=amd64
set GOBIN=D:\GO_Projects\bin
set GOCACHE=C:\Users\Nlaak\AppData\Local\go-build
set GOEXE=.exe
set GOHOSTARCH=amd64
set GOHOSTOS=windows
set GOOS=windows
set GOPATH=D:\GO_Projects
set GORACE=
set GOROOT=D:\Go
set GOTMPDIR=
set GOTOOLDIR=D:\Go\pkg\tool\windows_amd64
set GCCGO=gccgo
set CC=gcc
set CXX=g++
set CGO_ENABLED=1
set CGO_CFLAGS=-g -O2
set CGO_CPPFLAGS=
set CGO_CXXFLAGS=-g -O2
set CGO_FFLAGS=-g -O2
set CGO_LDFLAGS=-g -O2
set PKG_CONFIG=pkg-config
set GOGCCFLAGS=-m64 -mthreads -fmessage-length=0 -fdebug-prefix-map=C:\Users\Nlaak\AppData\Local\Temp\go-build597482594=/tmp/go-build -gno-record-gcc-switches

### Node Version
v8.11.3

### NPM Version
6.1.0

### Yarn Version
1.7.0

### PostgreSQL Version
PostgreSQL Not Found

### MySQL Version
MySQL Not Found

### SQLite Version
SQLite Not Found

### Dep Version
dep:
 version     : devel
 build date  :
 git hash    :
 go version  : go1.10.3
 go compiler : gc
 platform    : windows/amd64
 features    : ImportDuringSolve=false
</details>

**Comment by NlaakStudios** [comment]
I'm planning on taking care of this. However, I cannot assign it to myself or create a branch at this point.

**Comment by duffn** [comment]
Is this still something you are taking on @NlaakStudios?

**Comment by kteb** [comment]
It doesn't looks like it. @duffn I think if next week we don't have news from @NlaakStudios, you can ahead if you want.

**Comment by duffn** [comment]
:+1:

**Comment by NlaakStudios** [comment]
No, I am sorry, missed the messages until now. Ended up just writing my own Framework. You guys go ahead and sorry again for the delay.
